client/src/modules/employees/registry/search.modal.html

```diff
@@ -135,12 +144,34 @@
     </fieldset>
   </div>
 
-  <div class="modal-footer">
+
+
+
+      </uib-tab>
+
+       <uib-tab index="1" heading="{{ 'FORM.LABELS.DEFAULTS' | translate }}"  data-custom-filter-tab>
+
+         <div class="tab-body">
+
+
+          <div class="form-group" ng-class="{ 'has-error' : ModalForm.limit.$invalid }">
+            <label class="control-label" translate>FORM.LABELS.LIMIT</label>
+            <input name="limit" type="number" bh-integer class="form-control" ng-model="ModalCtrl.searchQueries.limit" ng-change="$ctrl.onSelectLimit($ctrl.defaultQueries.limit)">
```

Comment:
The standard practice is to make this model `ModalCtrl.defaultQueries.limit` instead of `ModalCtrl.searchQueries.limit`.  The idea is that we should be able to delete the `searchQueries` object and everything works properly, but the `defaultQueries` should always have some values set.

For example, see:
 1. [The Invoices Registry](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/invoices/registry/search.modal.html#L92)
 2. [The Patients Registry](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/patients/registry/search.modal.html#L118)
 3. [The Journal](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/modules/journal/modals/search.modal.html#L134)